### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Kybeer** [advplyr/audiobookshelf#818]
### Describe the issue

Hello,

I have Audiobookshelf server v. 2.0.23 running in a container (Podman). I'm using it exclusively with Android client 0.9.49-beta. 

I always keep the audiobook I currently listen to download:ed to the phone. 

It means the book is listed in the "local" on the main screen of the app. And when playing the book, it says "local" in green letters in the top. 

The problem I have is the app is using a lof of mobile data traffic, despite the book being downloaded. We're talked several MBs over 5-10 mintues of listening. Several 100MB with extensive listening. 
In other words, it's like the book is not being played locally, but over the internet/mobile data. 

This is not happening at home, when I'm on my local network (phone on wifi). It's happening when I'm out of home. 
I'm reaching my Audiobookshelf server over VPN when not at home. 

### Steps to reproduce the issue

1. Download the book in the Android app.
2. Leave my home or disconnect from wifi
3. Connect to my VPN
4. Listen to the book
5. Observe "mobile data usage" for Audiobookshelf app in Android. 

### Audiobookshelf version

2.0.23

### How are you running audiobookshelf?

Docker

**Comment by Kybeer** [comment]
Today I checked the traffic in my VPN app on Android, while listening to a downloaded book.

I use VPN to reach my server.

The amount of Rx+Tx MB keeps ticking up really fast - indicating the book is being streamed from my server - not read from the local file.

What's even more strange is that the TX uptick in the VPN app increased faster than RX 

There's no other traffic over the VPN app. If I pause the playback, the uptick stops.
